packages/flutter_tools/test/general.shard/base/error_handling_io_test.dart

```diff
@@ -114,6 +115,20 @@ void main() {
       when(mockFileSystem.path).thenReturn(MockPathContext());
     });
 
+    testWithoutContext('bypasses error handling when withAllowedFailure is used', () {
```

Comment:
The behavior of config is changing a bit, too, so it would be good to have a test of that as well.

Reply:
Done